openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -603,6 +625,141 @@ Location: https://client.example.net/cb?
   &error_description=Unsupported%20response_type%20value
 ```
 
+# Authorization Challenge Endpoint
```

Comment:
Discussing so much about nonces and challenges lately, I wonder if we have a better term than challenge here?

Reply:
As discussed on the call, this is the name used in the first-party apps draft.